cfme/tests/pod/test_appliance_crud.py

```diff
@@ -492,8 +588,36 @@ def test_aws_smartstate_pod():
         casecomponent: Containers
         caseimportance: medium
         initialEstimate: 1h
+        testSteps:
```

Comment:
This should be numbered - but isn't necessary for an automated test.  Please either remove, or number the list.